api-reference/20 Data Visualization Widgets/BaseChart/1 Configuration/onPointClick.md

```diff
@@ -57,7 +57,7 @@ The [onSeriesClick](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1
     import { Dx{WidgetName}Module } from "devextreme-angular";
     // ...
     export class AppComponent {
-        {widgetName}_onPointClick (e) {
+        onPointClick (e) {
```

Comment:
Let's rename this function into `cancelSeriesClick` in examples for all the technologies.